recipes-bsp/acpi-tables/files/edison/acpi-tables-load

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+# This script copies all ACPI tables into /kernel/firmware/acpi
+# You must have configfs enabled for this to work
+
+    for TABLE in `find /kernel/firmware/acpi/ -type f -name "*.aml" 2>/dev/null`; do
+        dest_dir=$(basename $TABLE .aml)
+        mkdir /sys/kernel/config/acpi/table/$dest_dir
+        cat $TABLE > /sys/kernel/config/acpi/table/$dest_dir/aml
+    done
+
+    gpioset `gpiofind TRI_STATE_ALL`=0
```

Comment:
As mentioned in the last patch I think better not to add these at all.

Reply:
I think if we take the 3rd patch, the 4th should be squashed with this one.

Reply:
As you request, I will drop 3rd and 4th patch. Instead I will put them in separate branch for future reference.